universal-login-react/src/ui/commons/Assets.tsx

```diff
@@ -23,7 +23,7 @@ export const Assets = ({deployedWallet}: AssetsProps) => {
       <div className="assets">
         <p className="assets-title">My Assets</p>
         <div className="assets-list">
-          {tokenDetailsWithBalance.map(({name, symbol, balance}: TokenDetailsWithBalance) => (
+          {(tokenDetailsWithBalance.filter((token) => !token.balance.isZero())).map(({name, symbol, balance}: TokenDetailsWithBalance) => (
```

Comment:
Extract to function and move to application layer

Reply:
Done